websites/serializers.py

```diff
@@ -209,6 +217,50 @@ def update(self, instance, validated_data):
         update_website_backend(instance.website)
         return instance
 
+    def get_content_context(self, instance):
+        """
+        Create mapping of uuid to a display name for any values in the metadata
+        """
+        if self.context and self.context.get("no_content_context"):
+            # Prevent circular dependencies where two relation fields may refer to one another
+            return None
```

Comment:
Can we add a test that creates this condition?

Reply:
We are testing this already: https://github.com/mitodl/ocw-studio/blob/gs/async-search-backend/websites/serializers_test.py#L170